observatory-platform/observatory/platform/docker/docker-compose.observatory.yml.jinja2

```diff
@@ -249,15 +266,15 @@ services:
       - discovery.type=single-node
 
   kibana:
-    image: "docker.elastic.co/kibana/kibana:7.9.0"
+    image: "docker.elastic.co/kibana/kibana:7.13.3"
     restart: always
     ports:
       - ${HOST_KIBANA_PORT}:5601
     networks:
       - {{ docker_network_name }}
 
   postgres:
-    image: postgres:12.2
+    image: postgres:13
```

Comment:
It would be good to stick with postgres:12.2 to avoid problems upgrading existing databases.

Reply:
reverting back to 12.2.  will template this out as part of installation script work.